src/Resources/Forms.php

```diff
@@ -165,4 +165,21 @@ public function getSubmissions($form_guid, $params = [])
 
         return $this->client->request('get', $endpoint, [], $queryString);
     }
+    
+    /**
+     * 
+     * Get a file uploaded via form by url
+     * 
+     * @param string $url
+     * 
+     * @see https://developers.hubspot.com/docs/methods/form-integrations/v1/uploaded-files/signed-url-redirect
+     */
+    public function getUploadedFileByUrl($url)
+    {
+        $parsed = explode('?', $url);
+        if (count($parsed) != 2) {
```

Comment:
Don't introduce any checks here, pass the value further, so client decides either to throw an exception or return an error response.